### ISSUE TITLE
Example queries return None 

### ISSUE THREAD

**Issue by latmarat** [mohanliu/qmpy_rester#5]
Hello, I have a question about qmpy_rester usage. I tried a few example queries from the Readme file but, whatever I try, I keep getting None as a result. Do you know what could be the issue? 
Thanks!

**Comment by mohanliu** [comment]
Yes. The reason is that our RESTful API for OQMD is still in the stage of internal test. We will release a new OQMD with RESTful API and some other features very soon. After that, you will be able to query data using this python wrapper. I will follow up with you once everything is ready.

**Comment by latmarat** [comment]
Thank you, looking forward to the new release!

**Comment by mohanliu** [comment]
The new OQMD is released! This API wrapper should be good now. Try it out and close the issue if everything looks fine.

**Comment by CompRhys** [comment]
From the query function:

```
def query_oqmd(i=0, offset=100):
    with qr.QMPYRester() as q:
        kwargs = {'limit':offset,'offset': i*offset, 'elements':'O'}
        # data = q.get_oqmd_phases(verbose=False, **kwargs)
        data = q.get_optimade_structures(verbose=False, **kwargs)
    return data["data"]
```

I have gotten the error:
```
File "/home/reag2/PhD/datasets/OQMD/script.py", line 13, in query_oqmd
    return data["data"]
TypeError: 'NoneType' object is not subscriptable
```

Is this a related issue?

**Comment by CompRhys** [comment]
> From the query function:
> 
> ```
> def query_oqmd(i=0, offset=100):
>     with qr.QMPYRester() as q:
>         kwargs = {'limit':offset,'offset': i*offset, 'elements':'O'}
>         # data = q.get_oqmd_phases(verbose=False, **kwargs)
>         data = q.get_optimade_structures(verbose=False, **kwargs)
>     return data["data"]
> ```
> 
> I have gotten the error:
> 
> ```
> File "/home/reag2/PhD/datasets/OQMD/script.py", line 13, in query_oqmd
>     return data["data"]
> TypeError: 'NoneType' object is not subscriptable
> ```
> 
> Is this a related issue?

This actually appears to be due to a gateway timeout. @mohanliu what are the access rules on making requests to the server? I had queried less than 3000 structures when it locked me out.

**Comment by tachyontraveler** [comment]
I increased the queue timeout on our web server now. I ran your script in my local system and it was able to download data from a query when I set the limit to be 3000. Lmk if this problem still persists.
